restcomm/restcomm.docs/sources-asciidoc/src/main/asciidoc/configuration/Starting Restcomm-Connect.adoc

```diff
@@ -20,13 +20,22 @@ where <xxx> is the release version number.
 *Restcomm-JBoss-AS7-7.6.0.869.zip*.
   Further we shall refer to this above Restcomm directory as the *$RESTCOMM_HOME*.
 
-*Step 2 – Configure Restcomm IP information and Text-to-speech*
+== Configure Restcomm IP information and Text-to-speech
 
 * Go to the directory *$RESTCOMM_HOME/bin/restcomm*.
 * Open the file *restcomm.conf*.
-* Go to the section *# Network configuration*.
+* Go to the section *# Network configuration*
+
+[source,bash]
+----
+# Network configuration
+NET_INTERFACE=''
+PRIVATE_IP=''
+SUBNET_MASK=''
+NETWORK=''
+BROADCAST_ADDRESS=''
+----
 
-image::images/7.png[]
 * In order to get the details of your server you should run the *ifconfig* command in the terminal.
 You will see something like this:
```

Comment:
can we enclose the command throughout the documents under [source,bash] as well ?